Lesson 5

petrovsky747 opened the pull request.

petrovsky747 committed:
Redesigned block "advantages" layout.
Added "footer" block.
Аdded the necessary fonts to the "fonts" folder for the "advantages" and "footer" blocks.

petrovsky747 committed:
Deleted reset.css

5tronciy commented on index.html:
```diff
@@ -1,45 +1,63 @@
+      <p class="footer_link">Careers</p>
+      <p class="footer_link">Privacy</p>
+      <p class="footer_link">Terms</p>
+      <p class="footer_copyright">© 2020 Triphouse, Inc. All rights reserved</p>
```
Why did you put it into the column? It looks separated in the design.
Also, you limited its width with the width of the column. Try to shrink the window to see.

petrovsky747 replied:
Would it be better to display these elements using a list?
Need to make the text in the column stretch in width when zooming out with a decrease in the distance between the column on the right?

5tronciy replied:
It doesn't make sense for me. But it is not a mistake to use a list here.
The text shouldn't be stretched. It is not clear what you gonna do. Look at copyright carefully - it is not on right place in your layout.

5tronciy commented on styles/styles.css:
```diff
@@ -22,29 +50,52 @@ table {
-    margin-right: 0;
+.advantages__subtitle {
+    margin-top: 48px;
+    margin-bottom: 122px;
```
It's your choice, but I prefer to add some margin-bottom to the wrapper to prevent sticking of any content to the next block.

petrovsky747 replied:
What value would be acceptable to add?

5tronciy replied:
It is 120px in the design and it is good to add it actually to the wrapper. Otherwise, you have to think about its content every time it changes.

5tronciy commented on styles/styles.css:
```diff
@@ -22,29 +50,52 @@ table {
+    font-size: 14px;
+    line-height: 1.2;
+    color: #FFFFFF;
+    margin: 163px auto 120px 0;
```
Its position depends on the content of first column.

5tronciy reviewed (changes requested):
Looks good.

petrovsky747 committed:
In the "advantages" block, the padding from the "subtitle" was removed, applied to the container.
For the layout of the footer, an unnumbered list is used.
The "copyright" element is moved outside the column, the margin from the list elements is configured.

5tronciy reviewed (approved):
Looks good
